### DISCUSSION TITLE
Can't generate the trace or topology graph

### DISCUSSION THREAD

**Discussion by alloevil** [apache/skywalking#7818] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/12680612/134903521-eceb9188-e39a-4ae5-8e1a-5f80ef533999.png)

My service runs in the docker container, and the dir looks like above, i add the agent config 
`"-javaagent:/home/work/skywalking-agent.jar","-Dskywalking.agent.service_name=mqs-device-unreleased-preview","-Dskywalking.collector.backend_service=10.38.163.209:11800",`
i checked the ui and there is no endpoints and topology like below
![image](https://user-images.githubusercontent.com/12680612/134904334-3ce57d27-6efb-4453-a6eb-b93af4085e42.png)
![image](https://user-images.githubusercontent.com/12680612/134904472-13978950-1efe-4c6b-9734-bb682a36e80e.png)

agent log
`[root@mqs-device-unreleased-prod-668b55b84b-5hrfs logs]# head -n 100 skywalking-api.log 
INFO 2021-09-27 15:16:37:799 main PluginBootstrap : no plugin files (skywalking-plugin.def) found, continue to start application. 
DEBUG 2021-09-27 15:16:38:820 SkywalkingAgent-7-ServiceManagementClient-0 ServiceManagementClient : ServiceManagementClient running, status:DISCONNECT. 
DEBUG 2021-09-27 15:16:38:830 SkywalkingAgent-12-GRPCChannelManager-0 GRPCChannelManager : Selected collector grpc service running, reconnect:true. 
DEBUG 2021-09-27 15:16:38:978 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.logging.InternalLoggerFactory : Using SLF4J as the default logging framework 
DEBUG 2021-09-27 15:16:38:981 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : -Dio.netty.noUnsafe: false 
DEBUG 2021-09-27 15:16:38:981 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : Java version: 8 
DEBUG 2021-09-27 15:16:38:983 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : sun.misc.Unsafe.theUnsafe: available 
DEBUG 2021-09-27 15:16:38:983 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : sun.misc.Unsafe.copyMemory: available 
DEBUG 2021-09-27 15:16:38:984 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : java.nio.Buffer.address: available 
DEBUG 2021-09-27 15:16:38:984 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : direct buffer constructor: available 
DEBUG 2021-09-27 15:16:38:985 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : java.nio.Bits.unaligned: available, true 
DEBUG 2021-09-27 15:16:38:986 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : jdk.internal.misc.Unsafe.allocateUninitializedArray(int): unavailable prior to Java9 
DEBUG 2021-09-27 15:16:38:986 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent0 : java.nio.DirectByteBuffer.<init>(long, int): available 
DEBUG 2021-09-27 15:16:38:986 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent : sun.misc.Unsafe: available 
DEBUG 2021-09-27 15:16:38:987 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent : -Dio.netty.tmpdir: /tmp (java.io.tmpdir) 
DEBUG 2021-09-27 15:16:38:987 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent : -Dio.netty.bitMode: 64 (sun.arch.data.model) 
DEBUG 2021-09-27 15:16:39:003 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent : -Dio.netty.maxDirectMemory: 10813440000 bytes 
DEBUG 2021-09-27 15:16:39:004 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent : -Dio.netty.uninitializedArrayAllocationThreshold: -1 
DEBUG 2021-09-27 15:16:39:005 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.CleanerJava6 : java.nio.ByteBuffer.cleaner(): available 
DEBUG 2021-09-27 15:16:39:005 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent : -Dio.netty.noPreferDirect: false 
DEBUG 2021-09-27 15:16:39:116 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.channel.MultithreadEventLoopGroup : -Dio.netty.eventLoopThreads: 8 
DEBUG 2021-09-27 15:16:39:151 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.InternalThreadLocalMap : -Dio.netty.threadLocalMap.stringBuilder.initialSize: 1024 
DEBUG 2021-09-27 15:16:39:151 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.InternalThreadLocalMap : -Dio.netty.threadLocalMap.stringBuilder.maxSize: 4096 
DEBUG 2021-09-27 15:16:39:159 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.channel.nio.NioEventLoop : -Dio.netty.noKeySetOptimization: false 
DEBUG 2021-09-27 15:16:39:159 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.channel.nio.NioEventLoop : -Dio.netty.selectorAutoRebuildThreshold: 512 
DEBUG 2021-09-27 15:16:39:170 SkywalkingAgent-12-GRPCChannelManager-0 org.apache.skywalking.apm.dependencies.io.netty.util.internal.PlatformDependent : org.jctools-core.MpscChunkedArrayQueue: available 
DEBUG 2021-09-27 15:16:39:422 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.ResourceLeakDetector : -Dorg.apache.skywalking.apm.dependencies.io.netty.leakDetection.level: simple 
DEBUG 2021-09-27 15:16:39:422 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.ResourceLeakDetector : -Dorg.apache.skywalking.apm.dependencies.io.netty.leakDetection.targetRecords: 4 
DEBUG 2021-09-27 15:16:39:431 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.AbstractByteBuf : -Dorg.apache.skywalking.apm.dependencies.io.netty.buffer.checkAccessible: true 
DEBUG 2021-09-27 15:16:39:431 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.AbstractByteBuf : -Dorg.apache.skywalking.apm.dependencies.io.netty.buffer.checkBounds: true 
DEBUG 2021-09-27 15:16:39:432 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.ResourceLeakDetectorFactory : Loaded default ResourceLeakDetector: org.apache.skywalking.apm.dependencies.io.netty.util.ResourceLeakDetector@a7b4487 
DEBUG 2021-09-27 15:16:39:492 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.numHeapArenas: 8 
DEBUG 2021-09-27 15:16:39:492 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.numDirectArenas: 8 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.pageSize: 8192 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.maxOrder: 11 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.chunkSize: 16777216 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.tinyCacheSize: 512 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.smallCacheSize: 256 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.normalCacheSize: 64 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.maxCachedBufferCapacity: 32768 
DEBUG 2021-09-27 15:16:39:493 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.cacheTrimInterval: 8192 
DEBUG 2021-09-27 15:16:39:494 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.cacheTrimIntervalMillis: 0 
DEBUG 2021-09-27 15:16:39:494 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.useCacheForAllThreads: true 
DEBUG 2021-09-27 15:16:39:494 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.PooledByteBufAllocator : -Dio.netty.allocator.maxCachedByteBuffersPerChunk: 1023 
DEBUG 2021-09-27 15:16:39:524 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.channel.DefaultChannelId : -Dio.netty.processId: 1 (auto-detected) 
DEBUG 2021-09-27 15:16:39:533 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.NetUtil : -Djava.net.preferIPv4Stack: false 
DEBUG 2021-09-27 15:16:39:534 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.NetUtil : -Djava.net.preferIPv6Addresses: false 
DEBUG 2021-09-27 15:16:39:535 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.NetUtil : Loopback interface: lo (lo, 127.0.0.1) 
DEBUG 2021-09-27 15:16:39:541 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.NetUtil : /proc/sys/net/core/somaxconn: 4096 
DEBUG 2021-09-27 15:16:39:543 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.channel.DefaultChannelId : -Dio.netty.machineId: 02:42:0a:ff:fe:77:86:8e (auto-detected) 
DEBUG 2021-09-27 15:16:39:574 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.ByteBufUtil : -Dio.netty.allocator.type: pooled 
DEBUG 2021-09-27 15:16:39:575 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.ByteBufUtil : -Dio.netty.threadLocalDirectBufferSize: 0 
DEBUG 2021-09-27 15:16:39:575 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.buffer.ByteBufUtil : -Dio.netty.maxThreadLocalCharBufferSize: 16384 
DEBUG 2021-09-27 15:16:39:601 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.Recycler : -Dio.netty.recycler.maxCapacityPerThread: 4096 
DEBUG 2021-09-27 15:16:39:601 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.Recycler : -Dio.netty.recycler.maxSharedCapacityFactor: 2 
DEBUG 2021-09-27 15:16:39:601 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.Recycler : -Dio.netty.recycler.linkCapacity: 16 
DEBUG 2021-09-27 15:16:39:601 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.Recycler : -Dio.netty.recycler.ratio: 8 
DEBUG 2021-09-27 15:16:39:601 grpc-default-executor-0 org.apache.skywalking.apm.dependencies.io.netty.util.Recycler : -Dio.netty.recycler.delayedQueue.ratio: 8 
DEBUG 2021-09-27 15:16:39:634 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND SETTINGS: ack=false settings={ENABLE_PUSH=0, MAX_CONCURRENT_STREAMS=0, INITIAL_WINDOW_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192} 
DEBUG 2021-09-27 15:16:39:645 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=983041 
DEBUG 2021-09-27 15:16:39:664 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND SETTINGS: ack=false settings={MAX_CONCURRENT_STREAMS=4, INITIAL_WINDOW_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192} 
DEBUG 2021-09-27 15:16:39:740 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND SETTINGS: ack=true 
DEBUG 2021-09-27 15:16:39:742 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=983041 
DEBUG 2021-09-27 15:16:39:743 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND SETTINGS: ack=true 
DEBUG 2021-09-27 15:16:39:759 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND HEADERS: streamId=3 headers=GrpcHttp2OutboundHeaders[:authority: 10.136.140.3:11800, :path: /skywalking.v3.EventService/collect, :method: POST, :scheme: http, content-type: application/grpc, te: trailers, user-agent: grpc-java-netty/1.32.1, agent-version: 8.6.0, grpc-accept-encoding: gzip, grpc-timeout: 29574030u] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false 
DEBUG 2021-09-27 15:16:39:777 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND DATA: streamId=3 padding=0 endStream=true length=597 bytes=00000002500a2438326466303963332d653932312d343230382d623865322d363538313837633833323664121c0a1a6d71732d6465766963652d756e72656c65... 
DEBUG 2021-09-27 15:16:39:781 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND PING: ack=false bytes=1234 
DEBUG 2021-09-27 15:16:39:781 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND PING: ack=true bytes=1234 
DEBUG 2021-09-27 15:16:39:786 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND HEADERS: streamId=3 headers=GrpcHttp2ResponseHeaders[:status: 200, content-type: application/grpc, grpc-encoding: identity, grpc-accept-encoding: gzip] padding=0 endStream=false 
DEBUG 2021-09-27 15:16:39:788 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND DATA: streamId=3 padding=0 endStream=false length=5 bytes=0000000000 
DEBUG 2021-09-27 15:16:39:788 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND PING: ack=false bytes=1234 
DEBUG 2021-09-27 15:16:39:792 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND HEADERS: streamId=3 headers=GrpcHttp2ResponseHeaders[grpc-status: 0] padding=0 endStream=true 
DEBUG 2021-09-27 15:16:39:794 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND PING: ack=true bytes=1234 
DEBUG 2021-09-27 15:16:39:845 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND HEADERS: streamId=5 headers=GrpcHttp2OutboundHeaders[:authority: 10.136.140.3:11800, :path: /skywalking.v3.JVMMetricReportService/collect, :method: POST, :scheme: http, content-type: application/grpc, te: trailers, user-agent: grpc-java-netty/1.32.1, agent-version: 8.6.0, grpc-accept-encoding: gzip, grpc-timeout: 29993727u] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false 
DEBUG 2021-09-27 15:16:39:847 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND DATA: streamId=5 padding=0 endStream=true length=521 bytes=00000002040acc0108a2bef3b0c22f12001a1a0801108080808028188080a0a42820b8aca9ec02288080e084251a151080809c0118808080f40220b88bf00628... 
DEBUG 2021-09-27 15:16:39:848 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND PING: ack=false bytes=1234 
DEBUG 2021-09-27 15:16:39:848 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND PING: ack=true bytes=1234 
DEBUG 2021-09-27 15:16:39:849 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND HEADERS: streamId=5 headers=GrpcHttp2ResponseHeaders[:status: 200, content-type: application/grpc, grpc-encoding: identity, grpc-accept-encoding: gzip] padding=0 endStream=false 
DEBUG 2021-09-27 15:16:39:849 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND DATA: streamId=5 padding=0 endStream=false length=5 bytes=0000000000 
DEBUG 2021-09-27 15:16:39:849 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND PING: ack=false bytes=1234 
DEBUG 2021-09-27 15:16:39:850 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND HEADERS: streamId=5 headers=GrpcHttp2ResponseHeaders[grpc-status: 0] padding=0 endStream=true 
DEBUG 2021-09-27 15:16:39:851 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND PING: ack=true bytes=1234 
DEBUG 2021-09-27 15:16:40:820 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND HEADERS: streamId=7 headers=GrpcHttp2OutboundHeaders[:authority: 10.136.140.3:11800, :path: /skywalking.v3.JVMMetricReportService/collect, :method: POST, :scheme: http, content-type: application/grpc, te: trailers, user-agent: grpc-java-netty/1.32.1, agent-version: 8.6.0, grpc-accept-encoding: gzip, grpc-timeout: 29999744u] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false 
DEBUG 2021-09-27 15:16:40:821 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] OUTBOUND DATA: streamId=7 padding=0 endStream=true length=306 bytes=000000012d0add0108f2cdf3b0c22f12091182609a077ef957401a1a0801108080808028188080e09c2d2098b5c7b405288080e09c2d1a151080809c01188080... 
DEBUG 2021-09-27 15:16:40:823 grpc-nio-worker-ELG-1-3 org.apache.skywalking.apm.dependencies.io.grpc.netty.NettyClientHandler : [id: 0x63d3014b, L:/10.119.134.142:57210 - R:/10.136.140.3:11800] INBOUND PING: ack=false bytes=1234 `

the logs shows INBOUND PING: ack=false, does it mean anything ?

**Comment by wu-sheng** [comment, marked as answer]
Do you deploy this on docker env? Make sure you Time Zone correct.

**Reply by alloevil** [reply]
My service runs on docker env and the skywalking backend service runs on the physical server.
The time zone are the correct.
This is the tz of my service runs on docker env below.
![image](https://user-images.githubusercontent.com/12680612/135009023-5b017978-c9f7-4e19-897f-3a46016337dd.png)
This is the tz of the skywalking backend service runs on physical server below.
![image](https://user-images.githubusercontent.com/12680612/135009319-956387f6-362b-4d21-8926-1969c5f72c16.png)
And the java version is 
`openjdk version "1.8.0_202"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_202-b08)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.202-b08, mixed mode)`

**Reply by wu-sheng** [reply]
Choose UTC0 on the UI. We have declared this in the first line of documents.

**Reply by alloevil** [reply]
Thanks for your reply, i will look into the doc.
